src/microbe_stage/editor/PatchMapNode.tscn

```diff
@@ -85,5 +87,15 @@ texture = ExtResource( 1 )
 expand = true
 stretch_mode = 6
 
+[node name="UnknownPatchLabel" type="Label" parent="MarginContainer/MarginContainer"]
```

Comment:
I think this needs the "PLACEHOLDER" editor text here to make sure the translation system doesn't pick this up. Or if you want to use the translation system then change the text to "UNKNOWN_PATCH_SYMBOL".